src/panels/config/automation/condition/types/ha-automation-condition-numeric_state.ts

```diff
@@ -20,15 +19,33 @@ export default class HaNumericStateCondition extends LitElement {
   }
 
   public render() {
-    const { value_template, entity_id, below, above } = this.condition;
+    const {
+      value_template,
+      entity_id,
+      attribute,
+      below,
+      above,
+    } = this.condition;
 
     return html`
       <ha-entity-picker
-        .value="${entity_id}"
-        @value-changed="${this._entityPicked}"
+        .value=${entity_id}
+        .name=${"entity_id"}
+        @value-changed=${this._valueChanged}
         .hass=${this.hass}
         allow-custom-entity
       ></ha-entity-picker>
+      <ha-entity-attribute-picker
+        .hass=${this.hass}
+        .entityId=${entity_id}
+        .value=${attribute}
+        .name=${"attribute"}
```

Comment:
```suggestion
        name="attribute"
```

Reply:
It should be set as property not as attribute.